### PR TITLE
feature: report submission

### PR DESCRIPTION
**Pull request by Asam237**
Report Submission

### CODE REVIEW

File: src/db/schema/report.schema.ts
Comment (monkeyK1n9) [inline src/db/schema/report.schema.ts:1]: RoadType will not more be an enum ?

**Review by monkeyK1n9 — CHANGES_REQUESTED** [review]
Review the road type ? who defines it ? is it an enum ? better define it both server and client side

File: src/db/schema/report.schema.ts
Comment (Asam237) [reply src/db/schema/report.schema.ts:1]: All rigth. i will do it.

**Review by monkeyK1n9 — APPROVED** [review]
Perfect

**Pull request merged**